comp/volunteers/marshals-training.yaml

```
@@ -1,6 +1,6 @@
 summary: Train marshals for the $SRYYYY game
 
-priority: minor
+priority: could
```

Trained marshals makes the matches (especially the early ones) run a lot smoother
```suggestion
priority: should
```